project-instructions.md

```diff
@@ -1,10 +1,13 @@
 For instructions on how to use this repository, consult the [official docs](https://docs.pagure.org/copr.copr/how_to_enable_repo.html#how-to-enable-repo).
 
-It should be enough to enable the copr repository for this project using the following command:
+We need a bit of post-configuration after enabling the copr repository for this project:
 
 ```
-$ dnf install 'dnf-command(copr)'
+$ dnf install -y jq envsubst 'dnf-command(copr)'
```

Comment:
Oh my

Reply:
Running `dnf -y install --skip-broken 'dnf-command(copr)' 'dnf5-command(copr)'`  would  do the trick, installing both dnf4 and dnf5 copr commands, and wouldn't fail in older systems where dnf5 did not exist.
That's fine for documented instructions. However, I'd avoid using that approach in an automated procedure as it would mask real failures :/

Reply:
@jchecahi I've incorporated those changes. Thank you.

Reply:
Nice, thanks! LGTM